core/stdclass.h

```diff
@@ -12,7 +12,11 @@
 #undef PAGE_MASK
 #define PAGE_MASK (PAGE_SIZE-1)
 #else
+#if defined(__APPLE__) && defined(__aarch64__)
```

Comment:
Bingo!
Although Apple A7 and A8 comes with 4KB page size by default, it was upgraded to 16KB also since iOS 8

Since all Apple's arm64 is using 64KB now, so fetching it dynamically may not be required as shown in 
https://reviews.llvm.org/rG1bebb2832ee312d3b0316dacff457a7a29435edb (line 201)